src/sentry/static/sentry/app/components/activity/note/mentionStyle.jsx

```diff
@@ -0,0 +1,74 @@
+import space from 'app/styles/space';
+
+/**
+ * Note this is an object for `react-mentions` component and
+ * not a styled component/emotion style
+ */
+export default function mentionStyle({minHeight} = {minHeight: 140}) {
```

Comment:
Need this as a function so we can specify different `minHeight`s (and maybe other props in the future).